src/icons.ts

```diff
@@ -27,6 +28,13 @@ export async function handleUploadIcon(root: string, req: any, res: any) {
 
       checkUploadIconType(files.file[0].path);
       checkFileSize(fileSize);
+      // the maximum icons number of each project is 10
+      // maybe users put the image files into the folder manually
+      while (!checkIconsNumber(root)) {
```

Comment:
We should create `projectIconFolder` if it is not created yet, otherwise `checkIconsNumber` would raise error.

Reply:
> We should create `projectIconFolder` if it is not created yet, otherwise `checkIconsNumber` would raise error.

Good catch, fixed. PTAL